src/upload.php

```diff
@@ -49,8 +51,18 @@
 $ext = pathinfo($oldFileName, PATHINFO_EXTENSION);
 $path = $newPath . '/' . $newFileName . '.' . $ext;
 
-// if directory for event's files does not exist, create it
+// file extention whitelist
+// to enable more file types just add the extentions with no . to the array.
+$allowedExtensions = array('txt', 'zip', 'pdf', 'docx', 'xlsx' ,'pptx');
+$isAllowed = in_array($ext, $allowedExtensions);
 
+if (!$isAllowed) {
+   $fileErrorMsg = "This file extion is not allowed.  Please upload a .txt, .zip,.pdf or an ms office file.";
```

Comment:
Typo with `extension`, though I'd probably just stay `type`. I would also be specific about allowed files:
```
This file type is not allowed. Accepted files: .txt, .zip, .pdf, .docx, .xlsx, .pptx
```